src/components/templates/Jazzbar/JazzTab/JazzTab.tsx

```diff
@@ -70,7 +70,7 @@ const Jazz: React.FC<JazzProps> = ({ setUserList, venue }) => {
 
   const jazzbarTables = venueToUse?.config?.tables ?? JAZZBAR_TABLES;
 
-  const { recentVenueUsers } = useRecentVenueUsers();
+  // const { recentVenueUsers } = useRecentVenueUsers();
```

Comment:
You can remove this since it's unused

Reply:
I don't know if this is needed in the nearest future and left a @debt comment

![image](https://user-images.githubusercontent.com/63194656/107030204-f232f000-67b8-11eb-891d-832557076ffb.png)

Reply:
Fair enough, lets leave it for now.